aliases/available/git.aliases.bash

```diff
@@ -45,6 +45,7 @@ alias gds='git diff --staged'
 alias gdt='git difftool'
 alias gdv='git diff -w "$@" | vim -R -'
 alias gc='git commit -v'
+alias gcf='git diff --name-only --diff-filter=U' # lists only conflicted files
```

Comment:
Greetings - Always great to see new git aliases.

Not thrilled with the `gc` prefix used here as its rather exclusively used for `commit` aliases.

This is a version of a `'list files'` action.

Although there aren't currently any aliases based on `'git ls-files'`, I think this action would live among them.

Q: Would '[git ls-files --unmerged](https://git-scm.com/docs/git-ls-files)' give the same (or similar-enough) results?

Although `'conflicted'` is technically correct, it seems `'unmerged'` is the official term.

I'm thinking `'glsu'` is the most appropriate alias for this action.

Reply:
@davidpfarrell I totally agree with you. I  have new proposal.  I would be cool to show the lines / blocks of the files that have conflicts

Basically we can use grep to show files starting with <<<<

Reply:
> Basically we can use grep to show files starting with <<<<

I don't know about that - I think listing the files that are unmerged is a good idea - I will definitely use the alias.

I just think that `glsu` is the best name for the alias.

Additionally, I think the alias could execute `git ls-file --unmerged` vs `git diff ...` (assuming they return similar results)